Code/GraphMol/catch_graphmol.cpp

```diff
@@ -3001,3 +3001,23 @@ TEST_CASE("github #4642: Enhanced Stereo is lost when using GetMolFrags") {
           Atom::ChiralType::CHI_UNSPECIFIED);
   }
 }
+
+TEST_CASE("Github #6119: No warning when merging explicit H query atoms with no bonds", "[bug][molops]"){
```

Comment:
Isn't the issue to _remove_ the warning, rather than add it to an additional case (at least based on the current issue title)?

Reply:
oh, shoot I understood this wrong. Thank you so much for pointing this out. I will fix it this weekend.

Reply:
Thank you for pointing that out

Reply:
I think the issue is actually that the warning *should* be issued in both cases: Hs are not merged in either case